libs/components/datetime/src/lib/modules/datepicker/calendar/daypicker-cell.component.ts

```diff
@@ -28,141 +29,94 @@ import { SkyDayPickerContext } from './daypicker-context';
   imports: [CommonModule, SkyDayPickerButtonComponent, SkyPopoverModule],
   selector: 'sky-daypicker-cell',
   standalone: true,
-  templateUrl: 'daypicker-cell.component.html',
+  templateUrl: './daypicker-cell.component.html',
 })
-export class SkyDayPickerCellComponent implements OnInit, OnDestroy {
+export class SkyDayPickerCellComponent {
+  readonly #calendarSvc = inject(SkyDatepickerCalendarService);
+  readonly #datepicker = inject(SkyDatepickerCalendarInnerComponent);
+
   /**
    * Whether the active date has been changed.
    */
-  @Input()
-  public activeDateHasChanged: boolean | undefined;
+  public activeDateHasChanged = input(false);
```

Comment:
Converting inputs and template properties to signals helped to address the reactivity issues the popovers were having.